internal/utils/env.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
I think this file should be removed. Please take a look at new style nsm applications:
https://github.com/networkservicemesh/cmd-nsmgr/blob/master/main.go
https://github.com/networkservicemesh/cmd-registry-proxy-dns/blob/master/main.go#L56
...